### DISCUSSION TITLE
Login no longer working?

### DISCUSSION THREAD

**Discussion by AllOnMyOhm** [YetiForceCompany/YetiForceCRM#14833] [How to]
Hello folks,

as of yesterday for some reason I'm completely unable to login to my installation of yetiforce, it doesn't redirect me from the login screen, it just sits there after I attempt to login.

Nothing was changed over the weekend, it just decided to no longer log me in... was there a recent update?

---  `Deleted, we don't publish private addresses to CRM` --

**Comment by mariuszkrzaczkowski** [comment]
if you have a problem with CRM, check the logs

**Reply by AllOnMyOhm** [reply]
I did indeed check the error log and there isn't anything new in there.
